### ISSUE TITLE
SCRIPTS ARE READY TO REVIEW

### ISSUE THREAD

**Issue by ldiazre** [ldiazre/testsim#1]
Hola Tomás, mis 2 scrpts están listos para revisión:
-ANALYSE_SAMPLE: Es el que prepara las muestras para el analisis
-RUN_PIPELINE: Es el que automatiza todo el proceso, y por tanto al que está anclado analyse_pipeline

GRACIAS

**Comment by tdido** [comment]
Hola Laura.

Los scripts funcionan perfecto. Un único detalle: la idea es generar el ambiente de conda manualmente, fuera de los scripts, y subir el archivo resultante (rna-seq.yaml) al repositorio. Pero bueno, más allá de ese cambio que podés hacer si querés, la pipeline funciona correctamente.

**Comment by ldiazre** [comment]
¿Te refieres a que dentro del mismo script generara una carpeta dentro de
"testsim" y que luego se subieran a git-hub?

**Comment by tdido** [comment]
No, simplemente que esos dos comandos no van dentro del script: se ejecutan a mano para generar el ambiente una única vez.

**Comment by ldiazre** [comment]
Hola de nuevo Tomás,
Respecto al examen del día 18, ¿será similar al del otro día? ¿o tendrá más
contenido?
Gracias

**Comment by tdido** [comment]
Ahí hice un post en el foro de la asignatura al respecto :)
